assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -13949,13 +13956,14 @@ public void doAttachUpload(RunData data, boolean singleFileUpload) {
                     // make an attachment resource for this URL
                     SecurityAdvisor sa = createSubmissionSecurityAdvisor();
                     try {
-                        String siteId = toolManager.getCurrentPlacement().getContext();
 
                         // add attachment
                         // put in a security advisor so we can create citationAdmin site without need
                         // of further permissions
                         securityService.pushAdvisor(sa);
+                        eventTrackingService.post(eventTrackingService.newEvent(AssignmentConstants.EVENT_ASSIGNMENT_SUBMISSION_ATTACHMENT_NEW1, assignmentReference+" "+name, siteId, true, NotificationService.NOTI_REQUIRED));
                         ContentResource attachment = contentHostingService.addAttachmentResource(resourceId, siteId, "Assignments", contentType, fileContentStream, props);
+                        eventTrackingService.post(eventTrackingService.newEvent(AssignmentConstants.EVENT_ASSIGNMENT_SUBMISSION_ATTACHMENT_NEW2, assignmentReference+" "+name, siteId, true, NotificationService.NOTI_REQUIRED));
```

Comment:
We reviewed this on the Core Call today. We agreed that the 2 "new" events is a a new pattern  we don't want to encourage (posting an event before and after a complicated or time consuming action). Instead, change these into log statements.

In this way, the information is still present in the logs for investigation purposes, but it doesn't introduce a new event prorogation pattern and continue to bloat the event system.